# Grupos de errores
error_groups:
  - name: "error_1"
    waypoints:
      - {x: 0.0, y: 0.0}
      - {x: 1.2, y: 0.0}

  - name: "error_2"
    waypoints:
      - {x: 0.0, y: 0.0}
      - {x: -1.2, y: 0.0}

  - name: "error_3"
    waypoints:
      - {x: 0.0, y: 0.0}
      - {x: 0.0, y: 1.2}

  - name: "error_4"
    waypoints:
      - {x: 0.0, y: 0.0}
      - {x: 0.0, y: -1.2}

  - name: "error_5"
    waypoints:
      - {x: 0.0, y: 0.0}
      - {x: 1.2, y: 1.2}

  - name: "error_6"
    waypoints:
      - {x: 0.0, y: 0.0}
      - {x: -1.2, y: -1.2}
